build.gradle

```diff
@@ -151,10 +151,15 @@ def versions = [
 dependencies {
   compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
 
+  compile group: 'com.microsoft.azure', name: 'azure-storage', version: '8.3.0'
```

Comment:
umm why are we using the really old client?
https://github.com/Azure/azure-sdk-for-java/tree/master/storage/data-plane

https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage-blob/11.0.1

Reply:
doesn't look like its used in this PR anyway?

Reply:
Because of this: https://github.com/Azure/azure-storage-java/issues/389

There should be a way apparently to get an input stream now, even from the new sdk, so at some point I'll see if the switch is a good idea.

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/master/storage/data-plane/src/main/java/com/microsoft/azure/storage/blob/TransferManager.java#L401

Reply:
I have seen it (see link I posted). I am not keen to use that without testing. v10 introduced many substantial changes and I am not that sure this is worth are use case. Btw for bulk scan we are still using the "old" version.

Reply:
We tried upgrading in bulk scan but it was a significant change, we said for new projects we'll use the new version but we won't bother upgrading bulk-scan as there isn't a big benefit.